lib/ansible/modules/database/postgresql/postgresql_publication.py

```diff
@@ -0,0 +1,653 @@
+      nothing will be changed. If you need to add all tables to the publication with the same name,
+      drop existent and create new without passing I(tables).
+    type: list
+  state:
+    description:
+    - The publication state.
+    default: present
+    choices: [ absent, present ]
+    type: str
+  parameters:
+    description:
+    - Dictionary with optional publication parameters.
+    - Available parameters depend on PostgreSQL version.
+    type: dict
+  owner:
+    description:
+    - Publication owner.
+    - If I(owner) is not defined, the owner will be set as I(login_user) or I(session_role).
+    type: str
+  cascade:
+    description:
+    - Drop publication dependencies. Has effect with I(state=absent) only.
+    type: bool
+notes:
+- PostgreSQL version must be 10 or greater.
+seealso:
+- name: CREATE PUBLICATION reference
+  description: Complete reference of the CREATE PUBLICATION command documentation.
+  link: https://www.postgresql.org/docs/current/sql-createpublication.html
+- name: ALTER PUBLICATION reference
+  description: Complete reference of the ALTER PUBLICATION command documentation.
+  link: https://www.postgresql.org/docs/current/sql-alterpublication.html
+- name: DROP PUBLICATION reference
+  description: Complete reference of the DROP PUBLICATION command documentation.
+  link: https://www.postgresql.org/docs/current/sql-droppublication.html
+author:
+- Loic Blot (@nerzhul) <loic.blot@unix-experience.fr>
+- Andrew Klychkov (@Andersson007) <aaklychkov@mail.ru>
+extends_documentation_fragment:
+- postgres
+'''
+
+EXAMPLES = r'''
+- name: Create a new publication with name "acme" targeting all tables in database "test"
+  postgresql_publication:
+    db: test
+    name: acme
+
+- name: Create publication "acme" publishing only prices and vehicles tables.
+  postgresql_publication:
+    name: acme
+    tables:
+    - prices
+    - vehicles
+
+- name: >
+    Create publication "acme", set user alice as an owner, targeting all tables.
+    Allowable DML operations are INSERT and UPDATE only
+  postgresql_publication:
+    name: acme
+    owner: alice
+    parameters:
+      publish: 'insert,update'
+
+- name: >
+    Assuming publication "acme" exists and there are targeted
+    tables "prices" and "vehicles", add table "stores" to the publication
+  postgresql_publication:
+    name: acme
+    tables:
+    - prices
+    - vehicles
+    - stores
+
+- name: Remove publication "acme" if exists in database "test"
+  postgresql_publication:
+    db: test
+    name: acme
+    state: absent
+'''
+
+RETURN = r'''
+exists:
+  description:
+  - Flag indicates the publication exists or not at the end of runtime.
+  returned: always
+  type: bool
+  sample: true
+queries:
+  description: List of executed queries.
+  returned: always
+  type: str
+  sample: [ 'DROP PUBLICATION "acme" CASCADE' ]
+owner:
+  description: Owner of the owner at the end of runtime.
```

Comment:
```suggestion
  description: Owner of the publication at the end of runtime.
```
?

Reply:
it is necessary to imagine that i'm another person (reviewer) and look at module's doc in detail... thanks!

Reply:
Even if you look at it in detail, these things are easy to overlook - when you wrote them yourself. This is why it's good to have someone else read docs as well ;)

Reply:
yep, good review(s) is more important than a PR itself!